content/10.diagnostics.md

```diff
@@ -247,14 +247,14 @@ This makes this approach a truly portable assay for point-of-care diagnostics.
 The authors achieved detection of 100 copies/&mu;l of pre-isolated RNA in 30 minutes, and correctly identified all SARS-CoV-2-positive patient RNA samples tested in 5 minutes (n=20).
 
 There is an increasing body of evidence that CRISPR-based assays offer a practical solution for rapid, low-barrier testing in areas that are at greater risk of infection, such as airports and local community hospitals.
-Technology based on CRISPR (clustered regularly interspaced short palindromic repeats) has also been instrumental in scaling up testing protocols.
+Technology based on CRISPR has also been instrumental in scaling up testing protocols.
```

Comment:
Does one of these need to be deleted? Repeated below

Reply:
yes! weird, I think this one is out of place!